backend/stakepoold/rpc/rpcserver/server.go

```diff
@@ -17,16 +17,21 @@
 package rpcserver
 
 import (
+	"time"
+
 	"golang.org/x/net/context"
 	"google.golang.org/grpc"
 
+	"github.com/decred/dcrd/chaincfg/chainhash"
 	pb "github.com/decred/dcrstakepool/backend/stakepoold/rpc/stakepoolrpc"
+	"github.com/decred/dcrstakepool/backend/stakepoold/userdata"
 )
 
 // Public API version constants
 const (
-	semverString = "3.0.0"
-	semverMajor  = 3
+	grpcTimeout  = time.Millisecond * 100
```

Comment:
Isn't that silly short?

Reply:
No it's just copying of overwriting a map which is ~2ms on my machine.  I think it could be re-written to not need a timeout.